.travis.yml

```diff
@@ -407,7 +275,7 @@ jobs:
       env: EXAMPLE_ENGINE=sqlite3
       before_install: skip
       install:
-        - pip install django~=2.0
+        - pip install django~=3.0
```

Comment:
Please allow this to remain the lowest possible version for ensuring the backwards-compatibility issue mentioned above, meaning `2.2`.